spec/models/tutorial_spec.rb

```diff
@@ -1,4 +1,15 @@
 require 'rails_helper'
 
 RSpec.describe Tutorial, type: :model do
+  describe 'class methods' do
+    it '.no_classroom_content' do
+      create(:tutorial, classroom: true)
+      create(:tutorial, classroom: true)
+      create(:tutorial)
+      create(:tutorial)
+      tutorials = Tutorial.no_classroom_content
+
+      expect(tutorials.count).to eq(2)
+    end
+  end
```

Comment:
could be a more specific test without too much effort.
```
      create(:tutorial, classroom: true)
      create(:tutorial, classroom: true)
      t1 = create(:tutorial)
      t2 = create(:tutorial)
      tutorials = Tutorial.no_classroom_content

      expect(tutorials.count).to eq(2)
      expect(tutorials).to include(t1)
      expect(tutorials).to include(t2)
```
or even
```
    expect(tutorials).to eq([t1,t2])
```

Reply:
Pushed up a new commit that has these changes in this test.